packages/core/src/components/forms/_common.scss

```diff
@@ -71,9 +71,9 @@ $control-group-stack: (
 @function dark-input-transition-shadow($color: $dark-input-shadow-color-focus, $focus: false) {
   @if $focus {
     @return
-      border-shadow(1, $color, 1px),
-      border-shadow(1, $color, 1px), // duplicating to minimize browser antialiasing in dark
-      border-shadow(0.3, $color, 3px);
+      inset border-shadow(1, $color, 1px),
```

Comment:
add inset focus border for dark theme and reduce outer border to 2px
![Screen Shot 2021-12-01 at 16 38 17](https://user-images.githubusercontent.com/5934779/144318154-1a290215-f683-45ab-bd10-7f0a9ab9d99b.png)